### DISCUSSION TITLE
Correct Usage of  "GRID_STAT_CLIMO_MEAN_FIELD"

### DISCUSSION THREAD

**Discussion by malloryprow** [dtcenter/METplus#1352] [Configuration] [answered]
I'm trying to use GRID_STAT_CLIMO_MEAN_FIELD to set the variable information for the climatology mean file. In my METplus conf I have it set as 

> GRID_STAT_CLIMO_MEAN_FIELD = { name="HGT"; level="P500"; }

but when I run METplus I get 

> 
> ERROR  :
> ERROR  : DictionaryEntry::dict_value() -> bad type
> ERROR  :

it looks like METplus is setting

> METPLUS_CLIMO_MEAN_DICT=climo_mean = {file_name = ["/gpfs/dell2/emc/verification/noscrub/emc.verif/global/climo/new_climo_files/era_interim/mean_0111"];field = ["{ name=\"HGT\"; level=\"P500\"; }"];regrid = {method = BILIN;width = 2;vld_thresh = 0.5;shape = SQUARE;}time_interp_method = NEAREST;match_month = FALSE;day_interval = 31;hour_interval = 6;}

**Comment by georgemccabe** [comment, marked as answer]
Hi @malloryprow , this was fixed in METplus 4.1.0-beta2. The change removes the extra quotation marks added around the field info. You will need to use this version or higher for GRID_STAT_CLIMO_MEAN_FIELD to work properly. If you must use an earlier version, you can likely set it by using GRID_STAT_MET_CONFIG_OVERRIDES. It should look something like this:

`GRID_STAT_MET_CONFIG_OVERRIDES = climo_mean = {field = [{ name="HGT"; level="P500"; }];}`

**Reply by georgemccabe** [reply]
Hi @malloryprow, were you able to get this to work using a beta version?

**Reply by malloryprow** [reply]
Oh yeah! I got it working! Sorry I didn't see your response!
